website/docs/kubernetes/viewing-and-selecting-current-kubernete-context.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 10
+title: Kubernetes context
+description: Viewing and selecting the current Kubernetes context
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes]
+tags: [migrating-to-kubernetes]
+---
+
+# Viewing and selecting the current Kubernetes context
+
+With Podman Desktop, you can view and select your current Kubernetes context.
+
+#### Prerequisites
+
+- You registered the Kubernetes cluster in your kubeconfig file: _<your_kubernetes_cluster>_. For example, [Creating a kind cluster](kind/creating-a-kind-cluster).
```

Comment:
This is a bit confusing? Couldn't we say something like: "You have a Kubernetes context configured within `~/.kube/config` ?